hyperledger_matriculation_api.md

```diff
@@ -129,8 +129,7 @@ This method adds a single entry to the list of semesters in the MatriculationDat
 - ID = addEntryToMatriculationData
```

Comment:
```suggestion
- ID = addEntriesToMatriculationData
```

Considering that we now add multiple entries, the transaction should be named accordingly